### PR TITLE
Better chapter titles

### PR DESCRIPTION
**Pull request by ingydotnet**
Make chapter titles more accurate.

By submitting this pull request, I confirm that you can use, modify, copy, and
redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: www/main/review/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 <meta http-equiv="Content-Type"
       content="text/html; charset=UTF-8" />
 
-<title>v1 Review YAML Spec Conversion</title>
+<title>Markdown / 1.2.2 / 1.2.1 Compare</title>
```
Comment (Thom1729) [inline www/main/review/index.html:8]: Can you summarize what the changes to this file are about?

File: www/main/review/index.html [hunk 2]
```diff
@@ -105,14 +106,27 @@
...
+  mmap['# Chapter #. Flow Style Productions']   = 'H1 Chapter 7.'
+  mmap['# Chapter #. Block Style Productions']  = 'H1 Chapter 8.'
+  mmap['# Chapter #. Document Stream Productions'] = 'H1 Chapter 9.'
+  mmap['# Chapter #. Recommended Schemas']      = 'H1 Chapter 10.'
+
   io = new IO (e)->
     return unless e[0].isIntersecting
     h = $(e[0].target).text()
-    if h.match /^(#{1,4})\s/
+    console.log(h)
```
Comment (Thom1729) [inline www/main/review/index.html:123]: Left in by mistake?

File: www/main/review/index.html (hunk 1 above)
Comment (ingydotnet) [reply www/main/review/index.html:8]: It's the title of this web page: https://spec.yaml.io/194/review/

File: www/main/review/index.html (hunk 2 above)
Comment (ingydotnet) [reply www/main/review/index.html]: fixed. thanks.

**Review by Thom1729 — APPROVED** [review]

**Review by perlpunk — COMMENTED** [review]
For the record, I'm still opposed to title changes
